endoreg_db/utils/storage.py

```
@@ -30,7 +30,7 @@ def file_exists(field_file: Optional[FieldFile]) -> bool:
     if not _has_field_file(field_file):
         return False
     try:
-        return field_file.storage.exists(field_file.name)  # type: ignore[union-attr]
+        return field_file.storage.exists(field_file.name)[union - attr]
```

Pylint (4.0.2)

[error] Undefined variable 'union'

(E0602)